### ISSUE TITLE
Is it possible to reduce launch time?

### ISSUE THREAD

**Issue by plonguini** [otamajakusi/dockerfile-yolov5-jetson#12]
Hi @otamajakusi, thank you for this tutorial. It is very helpful.  I was wondering if it is possible to reduce the launch time (currently taking around 5 minutes on my Jetson Nano 2gb)? Also, is it possible to run a different version of the detect.py (stored locally) in the same way when running with our own weights?

Thanks and regards!

**Comment by otamajakusi** [comment]
@plonguini 
Please try. This can reduce the time for downloading weights.
```
xhost +local:
docker run -it --rm \
           --runtime nvidia \
           --network host \
           --device /dev/video0:/dev/video0:mrw \
           -e DISPLAY=$DISPLAY \
           -v /tmp/.X11-unix/:/tmp/.X11-unix \
           -v /path/to/yolov5-stored-locally:/yolov5 \
           yolov5 python3 detect.py --source 0 --weights /yolov5/your-weights-stored-locally.pt
```
Your weights file `your-weights-stored-locally.pt`, should be placed on your yolov5 stored locally.

**Comment by plonguini** [comment]
Hi @otamajakusi, thank you for your quick reply. So, today the first launch took a little less than 3 minutes. I guess some memory was freed when the nano rebooted, idk...

Then I tried running with my weights stored inside yolov5, and got the launch time down to around 2:40s. Then I tried not removing the container. Starting it up again took around 2:48s. Lastly I tried launching a new container with my own weights stored locally but without the display option, and it took 2:32s to launch.

So my question is (maybe a dumb one, but still): are these launch times due to the containers solution or would we be facing such long launch times even if it was ran on the host environment of the Nano? I'm asking because launching yolov5 from an old notebook I have (i5 dual core 2.3ghz) takes only a few seconds...

Thank you very much!
